text/accumulation.tex

```diff
@@ -30,118 +34,206 @@ \subsection{Preimage Integration}
+\begin{equation}
+  \label{eq:partialstate}
+  \partialstate \equiv \left(\begin{aligned}
+    &\isa{\mathbf{d}}{\dict{\N_S}{\mathbb{A}}} \,,\;
+    \isa{\mathbf{i}}{\seq{\mathbb{K}}_\mathsf{V}} \,,\;
+    \isa{\mathbf{q}}{\seq{\seq{\H}_\mathsf{Q}}_\mathsf{C}} \,,\\
+    &\isa{\mathbf{x}}{(\N_S,\N_S,\N_S,\dict{\N_S}{\N_G})}
+  \end{aligned}\right)
+\end{equation}
+
+We denote the set characterizing a \emph{deferred transfer} as $\defxfer$, noting that a transfer includes a memo component $m$ of $\mathsf{M} = 128$ octets, together with the service index of the sender $s$, the service index of the receiver $d$, the amount of tokens to be transferred $a$ and the gas limit $g$ for the transfer. Formally:
 \begin{align}
-  A \colon \left\{\begin{aligned}
-      \N_S &\to \ltuple\ \begin{aligned}
-        \isa{\mathbf{s}&}{\mathbb{A}\bm{?}}\ts\quad
-        \isa{\mathbf{v}}{\seq{\mathbb{K}}_\mathsf{V}}\ts\quad
-        \isa{\mathbf{t}}{\seq{\mathbb{T}}}\ts\quad
-        \isa{r}{\H\bm{?}}\ts\\
-        \isa{\mathbf{c}&}{\lseq\lseq\H\rseq_\mathsf{Q}\rseq_\mathsf{C}}\ts\qquad
-        \isa{\mathbf{n}}{\dict{\N_S}{\mathbb{A}}}\ts\\
-        \isa{p&}{\tuple{\isa{m}{\N_S}, \isa{a}{\N_S}, \isa{v}{\N_S}, \isa{\mathbf{g}}{\dict{\N_S}{\N_G}}}}\\
-      \end{aligned}\rtuple \\
-      s &\mapsto \Psi_A(\delta^\dagger, s, G(s) + \subifnone((\chi_\mathbf{g})_s, 0), M(s))
-    \end{aligned}\right.
+  \defxfer \equiv \ltuple\isa{s}{\N_S}\ts\isa{d}{\N_S}\ts\isa{a}{\N_B}\ts\isa{m}{\Y_{\mathsf{M}}}\ts\isa{g}{\N_G}\rtuple
 \end{align}
 
-As can be seen plainly, our accumulation mapping $A$ combines portions of the prior state into arguments for a virtual-machine invocation. Specifically the service accounts $\delta^\dagger$ together with the index of the service in question $s$ and its gas limit and wrangled refine-results $M(s)$ are arranged to create the arguments for $\Psi_A$, itself using a virtual-machine invocation as defined in appendix \ref{sec:accumulateinvocation}. Note that the gas limit is the sum of the regular gas $G(s)$ together with any privileged gas it receives $(\chi_\mathbf{g})_s$.
+Finally, we denote the set of service/hash pairs, utilized as a service-indexed commitment to the accumulation output, as $\servouts$:
+\begin{equation}
+  \servouts \equiv \{(\N_S, \H)\}
+\end{equation}
+
+We define the outer accumulation function $\Delta_+$ which transforms a gas-limit, a sequence of work-reports, an initial partial-state and a dictionary of services enjoying free accumulation, into a tuple of the total gas utilized in \textsc{pvm} execution $u$, a posterior state-context $(\mathbf{x}', \mathbf{d}', \mathbf{i}', \mathbf{q}')$ and the resultant deferred-transfers $\defxfer$ and accumulation-output pairings $\servouts$:
```

Comment:
The names u, x', d', etc don't seem to be used below any more, so should probably be removed here?